recipes-connectivity/networkmanager/networkmanager/firstboot-networkmanager-setup

```diff
@@ -28,6 +28,11 @@ done
 systemctl disable dnsmasq
 systemctl stop dnsmasq
 
+rm /etc/resolv.conf
```

Comment:
Is not better keep all the workarrounds in here than spread it into .bbappend of systemd and dnsmasq? This way if in future we want to go back to connman(after someone adds support to our modem in ofono) it would be just remove the nm and mm from image and add comman and ofono.